api/src/Common/Models/Organization.cs

```diff
@@ -10,6 +10,12 @@ public struct Organization : IModel
         [Required]
         public string Name { get; set; }
 
+        [Required]
+        public string WildApricotDomain { get; set; }
```

Comment:
I remember we decided to put these values in the organization class but I don't remember why. This limits us to using wild apricot with all orgs given that it's a required field. What was our reasoning?

Reply:
1. We need to know on the frontend the domain and client secret to navigate the user to for logging in
2. Because of this, we can either hard code ESR and it's clientId everywhere, or put it on the model. Model is definitely better.